### PR TITLE
Adding Quick Install config file

### PR DESCRIPTION
**Pull request by bsantacruzms**
Adding Quick install config file.

### CODE REVIEW

File: config-34.tpl.yml [hunk 1]
```diff
@@ -0,0 +1,500 @@
+---
```
Comment (xpillons) [inline config-34.tpl.yml:1]: Why having another template config file ?

File: config-qinstall.yml [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+  enforce: false
+#   grant_access_from: [a.b.c.d] # Array of CIDR to grant access from, see https://docs.microsoft.com/en-us/azure/storage/common/storage-network-security?tabs=azure-portal#grant-access-from-an-internet-ip-range
+  public_ip: true # Enable public IP creation for Jumpbox, OnDemand and create images. Default to true
+
+# Base image configuration. Can be either an image reference or an image_id from the image registry or a custom managed image
+linux_base_image: "OpenLogic:CentOS:7_9-gen2:latest" # publisher:offer:sku:version or image_id
+# linux image plan if required, format is publisher:product:name
+#linux_base_plan:
+windows_base_image: "MicrosoftWindowsServer:WindowsServer:2019-Datacenter-smalldisk:latest" # publisher:offer:sku:version or image_id
+lustre_base_image: "azhpc:azurehpc-lustre:azurehpc-lustre-2_12:latest"
```
Comment (xpillons) [inline config-qinstall.yml:96]: both lustre properties should be removed

File: config-qinstall.yml [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+# Default is basic authentication. For oidc authentication you have to specify the following values
+# The OIDCClient secret need to be stored as a secret named <oidc-client-id>-password in the keyvault used by az-hop
+authentication:
+  httpd_auth: basic # oidc or basic
+
+image_gallery:
+  create: true # Create the shared image gallery to store custom images
+
+# List of images to be defined
+images:
```
Comment (xpillons) [inline config-qinstall.yml:174]: I guess the whole image section could be removed for simplicity

File: config-qinstall.yml [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+    max_core_count: 200
+    image: azhpc:azhop-desktop:centos-7_9:latest
+    ColocateNodes: false
+    spot: false
+    EnableAcceleratedNetworking: true
+    max_hours: 12
+    min_hours: 1
+
+# Remote Visualization definitions
+enable_remote_winviz: false # Set to true to enable windows remote visualization
```
Comment (xpillons) [inline config-qinstall.yml:328]: enable_remote_winviz could be removed

File: config-qinstall.yml [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+
+remoteviz:
```
Comment (xpillons) [inline config-qinstall.yml:330]: remoteviz could be removed

File: config-qinstall.yml [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+    enabled: true
+  bc_amlsdk:
+    enabled: false
+  bc_rstudio:
+    enabled: false
+  bc_ansys_workbench:
+    enabled: false
+  bc_vmd:
+    enabled: false
+  bc_paraview:
```
Comment (xpillons) [inline config-qinstall.yml:353]: I would enabled paraview as it's used with the OpenFOAM tutorial

File: docs/tutorials/quick_install.md [hunk 7]
```diff
@@ -170,13 +131,14 @@ After this the WSL2 is ready to use.
 
 During this task, you will prepare the **config.yml** file to configure the environment.
 
-1. Get into the az-hop folder downloaded previously on Task #2 and execute the following command to create a new copy of the file with the right extension.
+1. Get into the az-hop folder downloaded previously on Task #2 and execute the following command to create a new copy of the config file with the right extension.
+    >Note: During this exercise you will use a lighter version of the original config file designed for the Quick Install guide.
 
    ```bash
-   cp config.tpl.yml config.yml
+   cp config-qinstall.yml config.yml
```
Comment (xpillons) [inline docs/tutorials/quick_install.md:138]: Change path from examples

File: config-qinstall.yml (hunk 2 above)
Comment (bsantacruzms) [reply config-qinstall.yml:96]: Removed

File: config-qinstall.yml (hunk 3 above)
Comment (bsantacruzms) [reply config-qinstall.yml:174]: Removed

File: config-qinstall.yml (hunk 4 above)
Comment (bsantacruzms) [reply config-qinstall.yml:328]: Removed

File: config-qinstall.yml (hunk 6 above)
Comment (bsantacruzms) [reply config-qinstall.yml:353]: Enabled

File: docs/tutorials/quick_install.md (hunk 7 above)
Comment (bsantacruzms) [reply docs/tutorials/quick_install.md:138]: Created path. Tutorials>quickinstall>config.yml

File: config-34.tpl.yml [hunk 8]
```diff
@@ -0,0 +1,500 @@
...
+# yaml-language-server: $schema=config.schema.json
```
Comment (xpillons) [inline config-34.tpl.yml:2]: Please remove file config-34.tpl.yml

File: examples/index.md [hunk 9]
```diff
@@ -5,7 +5,10 @@ This folder contains examples of pre-defined configuration files for common scen
...
 |--------------------------|-----------|---------|-------------|---------------|----------|--------|------------------|
 | minimum_public_ip.yml    | Yes       | ANF     | Create      | No            | No       | No     | No               |
-
+| qinstall-config.yml    | Yes       | ANF     | Create      | No            | No       | No     | No               |
 
 ## minimum_public_ip.yml
 This configuration file is defining the minimum required to build an `azhop` environment when public IP is allowed.
+
+## qinstall-config.yml
+This configuration file is defining the minimum required to quickly deploy `azhop` environment.
```
Comment (xpillons) [inline examples/index.md:14]: Please describe that it comes with the quick install or add a link to the tutorial page

File: examples/qinstall-config.yml [hunk 10]
```diff
@@ -0,0 +1,264 @@
...
+  enroot_version: 3.4.1
+
+# Authentication configuration for accessing the az-hop portal
+# Default is basic authentication. For oidc authentication you have to specify the following values
+# The OIDCClient secret need to be stored as a secret named <oidc-client-id>-password in the keyvault used by az-hop
+authentication:
+  httpd_auth: basic # oidc or basic
+
+image_gallery:
+  create: true # Create the shared image gallery to store custom images
```
Comment (xpillons) [inline examples/qinstall-config.yml:168]: We don't need an image gallery as we are using marketplace images. You should be able to remove the whole section

File: examples/qinstall-config.yml [hunk 11]
```diff
@@ -0,0 +1,264 @@
...
+# On demand VM configuration
+ondemand:
+  vm_size: Standard_D4s_v5
+  #fqdn: azhop.foo.com # When provided it will be used for the certificate server name
+  generate_certificate: true # Generate an SSL certificate for the OnDemand portal. Default to true
+# Grafana VM configuration
+grafana:
+  vm_size: Standard_B2ms
+# Guacamole VM configuration
+guacamole:
```
Comment (xpillons) [inline examples/qinstall-config.yml:117]: guacamole should be removed

File: examples/qinstall-config.yml [hunk 12]
```diff
@@ -0,0 +1,264 @@
...
+# Jumpbox VM configuration, only needed when deploying thru a public IP and without a configured deployer VM
+jumpbox:
+  vm_size: Standard_B2ms
+  # SSH port under which the jumpbox SSH server listens on the public IP. Default to 22
+  # Change this to, e.g., 2222, if security policies (like "zero trust") in your tenant automatically block access to port 22 from the internet
+  #ssh_port: 2222
+# Active directory VM configuration
+ad:
+  vm_size: Standard_B2ms
+  hybrid_benefit: false # Enable hybrid benefit for AD, default to false
```
Comment (xpillons) [inline examples/qinstall-config.yml:106]: for simplicity, hubrid_benefit and high_availability can be removed

File: examples/qinstall-config.yml [hunk 13]
```diff
@@ -0,0 +1,264 @@
...
+        create: true
+      compute:
+        name: compute
+        address_prefixes: "10.0.1.0/24"
+        create: true
+  # Specify the Application Security Groups mapping if already existing
+
+# When working in a locked down network, uncomment and fill out this section
+locked_down_network:
+  enforce: false
```
Comment (xpillons) [inline examples/qinstall-config.yml:87]: for simplicity enforce could be removed

File: examples/qinstall-config.yml [hunk 14]
```diff
@@ -0,0 +1,264 @@
...
+    server: '{{anf_home_ip}}' # Specify an existing NFS server name or IP, when using the ANF built in use '{{anf_home_ip}}'
+    export: '{{anf_home_path}}' # Specify an existing NFS export directory, when using the ANF built in use '{{anf_home_path}}'
+    options: '{{anf_home_opts}}' # Specify the mount options. Default to rw,hard,rsize=262144,wsize=262144,vers=3,tcp,_netdev
+
+# name of the admin account
+admin_user: hpcadmin
+# List of identities (object ids) to grant read access to az-hop key vault (optional)
+# key_vault_readers:
+# Network
+network:
```
Comment (xpillons) [inline examples/qinstall-config.yml:52]: to simplify the network section, remove create_nsg, then all the name and the create properties the subnets

File: docs/tutorials/quick_install.md [hunk 15]
```diff
@@ -187,15 +148,15 @@ During this task, you will prepare the **config.yml** file to configure the envi
 
 1. Run the Terraform plan script to validate changes to be executed.
     ```bash
-    ./build.sh -a plan
+   ./build.sh -a plan
     ```
     If no errors have been detected you can proceed to execute the following code to start the deployment.
     ```bash
-    ./build.sh -a apply 
+   ./build.sh -a apply
```
Comment (xpillons) [inline docs/tutorials/quick_install.md:155]: shouldn't use bicep here ? if so please add "-l bicep"

File: docs/tutorials/quick_install.md [hunk 16]
```diff
@@ -187,15 +148,15 @@ During this task, you will prepare the **config.yml** file to configure the envi
 
 1. Run the Terraform plan script to validate changes to be executed.
```
Comment (xpillons) [inline docs/tutorials/quick_install.md:149]: if we use bicep then this would be useless

**Review by xpillons — CHANGES_REQUESTED** [review]

File: config-34.tpl.yml (hunk 8 above)
Comment (bsantacruzms) [reply config-34.tpl.yml:2]: Done

File: examples/index.md (hunk 9 above)
Comment (bsantacruzms) [reply examples/index.md:14]: I added the link to follow the guide.

File: examples/qinstall-config.yml (hunk 10 above)
Comment (bsantacruzms) [reply examples/qinstall-config.yml:168]: Removing image gallery.

File: examples/qinstall-config.yml (hunk 11 above)
Comment (bsantacruzms) [reply examples/qinstall-config.yml:117]: Removed

File: examples/qinstall-config.yml (hunk 12 above)
Comment (bsantacruzms) [reply examples/qinstall-config.yml:106]: Removed

File: docs/tutorials/quick_install.md (hunk 15 above)
Comment (bsantacruzms) [reply docs/tutorials/quick_install.md:155]: Yes, I added the parameter now.

File: docs/tutorials/quick_install.md (hunk 16 above)
Comment (bsantacruzms) [reply docs/tutorials/quick_install.md:149]: I updated the command to ./build.sh -a plan -l bicep

File: examples/qinstall-config.yml (hunk 14 above)
Comment (bsantacruzms) [reply examples/qinstall-config.yml:52]: Removing those lines as well.

**Comment by bsantacruzms** [comment]
@xpillons I have finished to test the quick install with Bicep instructions. You can go ahead and merge this.

**Pull request merged**